### ISSUE TITLE
Exclude datasets starting with 'pdf-meta' from count

### ISSUE THREAD

**Issue by cole-fields** [genkimaps/gis-hub#120]
Hiding the pdf-meta from display... they are still counted as datasets, just not displayed on /dataset page.
 - Dataset count is misleading/incorrect
 - I did a batch dataset patch for all the pdf-meta records and then they're displayed first in the query results. so there are pages of empty/hidden datasets until user actually gets to populated/visible dataset records

Related issue: https://github.com/genkimaps/gis-hub/issues/110

**Comment by cole-fields** [comment]
@genkimaps just a note, that I added a filter to the before_search method in our dfo extension. see commit for details, but it makes the count of datasets more accurate by ignoring the pdf-metadata records. took a while to figure out and troubleshoot, but i think it's better than how i was doing it before (just not showing the pdf metadata datasets on the /datasets page.

**Comment by cole-fields** [comment]
This seems to be working correctly now. The original commit I tried (and thought worked) was applying a query against the datasets would disregard the filter queries (like filter by organization or group) and was then returning all the datasets on the site (other than the ignored ones). 

Changed it to a filter query (fq) that is appended to any existing fq to ensure than they will work when combimed.
